WordPress/Classes/Networking/TaxonomyServiceRemoteREST.h

```diff
@@ -2,6 +2,20 @@
 #import "TaxonomyServiceRemote.h"
 #import "SiteServiceRemoteREST.h"
 
+extern NSString * const TaxonomyRESTCategoryIdentifier;
+extern NSString * const TaxonomyRESTTagIdentifier;
+
+extern NSString * const TaxonomyRESTIDParameter;
```

Comment:
I was originally thinking this but it felt wrong retyping all of the strings. However, I think it makes sense to do so as we are indeed testing for any unexpected changes.

I've updated a commit to use explicit strings instead of consts.